http/dashboard_service.go

```diff
@@ -472,12 +472,33 @@ func (h *DashboardHandler) handleGetDashboard(w http.ResponseWriter, r *http.Req
 		return
 	}
 
+	include := r.URL.Query().Get("include")
```

Comment:
could one-line this into `showViewProperties := r.URL.Query().Get("include") == "properties"`